src/api.js

```diff
@@ -0,0 +1,3 @@
+import request from 'superagent';
+
+export const getCharacter = (pokemonId) => request.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex?name=${pokemonId}`)
```

Comment:
name=${pokemonId} should be pokemon=${pokemonId}. This is the only thing causing your detail page to not render